openapi/components/schemas/APIConfig.yaml

```diff
@@ -428,50 +213,23 @@ properties:
     type: string
   hubWorldId:
     $ref: ./WorldID.yaml
+  imageHostUrlList:
+    description: Allowed hosts, where string data can be loaded from.
```

Comment:
This description isn't quite accurate, we should probably reference the Udon documentation for https://creators.vrchat.com/worlds/udon/image-loading.

Reply:
What do you want to be there?
something like `see https://creators.vrchat.com/worlds/udon/image-loading/`?

Reply:
Something like:

A list of explicitly allowed origins that worlds can request images from via the Udon's [VRCImageDownloader#DownloadImage](https://creators.vrchat.com/worlds/udon/image-loading/#downloadimage).